cli/src/cli/scaffold/rust.rs

```diff
@@ -79,7 +78,7 @@ fn interpolate_cargo_template(
 
 impl RustScaffold {
     pub fn new(package_name: &str, macro_style: HdkMacroStyle) -> RustScaffold {
-        let target_dir = wasm_target_dir(&package_name.into(), &String::new().into());
+        let target_dir = PathBuf::from("../target");
```

Comment:
I don't quite understand why this was the way it was. Using `wasm_target_dir` would resulted in the build artifacts being given absolute path (non-portable). Does it not run on CI this way perhaps?